src/client.cpp

```diff
@@ -0,0 +1,22 @@
+//
+// Created by Illia Plaksa on 25.01.2022.
+//
+
+#include "controller/RemoteController.h"
+#include "cli/include/StepMachine.h"
+
+int main(int argc, char** argv)
+{
+    auto channel = grpc::CreateChannel("localhost:50051", grpc::InsecureChannelCredentials());
+    auto stub = service::RequestHandler::NewStub(channel);
+
+    auto controller = std::make_shared<RemoteController>(std::move(stub));
+
+    auto console_manipulator = std::make_shared<ConsoleManipulator>(std::cout, std::cin);
```

Comment:
ConsoleManipulator could be renamed to StreamManipulator, as apparently it works with arbitrary i/o streams